osc.c

```diff
@@ -1075,6 +1075,11 @@ static void load_plugins(GtkWidget *notebook, const char *ini_fn)
 #endif
 		snprintf(buf, sizeof(buf), "%s/%s", plugin_dir, ent->d_name);
 
+		/* Don't load obsolete plugins to avoid any potential issues */
+		if (!strncmp("fmcomms1.", ent->d_name, strlen("fmcomms1."))) {
+			continue;
+		}
```

Comment:
We are not sure. But my best guess is that there aren't because the hardware is really old (10 years+) and fmcomms2 is by far an upgrade to fmcomms1.
I am looking at the commit history for file fmcomms1.c and the last 17 commits are all about maintaining code (removing deprecated methods, fixing mem leaks, fixes to remove compiler warnings - so nothing that indicates the hardware was used and a fix was made, or an enhancement was added).
At this moment, we're trying to get rid of gdk_threads_enter/leave() across osc and plugins and this plugin has several such calls. So I would like to put a stop to the series of 17 maintenance commits.
If someone is still using fmcomms1 we will likely hear about that after this commit or next osc release and we will know it's worthed to add it back.

Reply:
Alright...